### DISCUSSION TITLE
View statistics per post type

### DISCUSSION THREAD

**Discussion by frietboer** [ibericode/koko-analytics#255] [Ideas]
Is there any way of displaying statistics per post type? I would have used the "Most Viewed Posts" shortcode, but it does not display the actual amounts of views. Is this something that could be added easily? Or maybe add the post type in the dashboard listings?

**Comment by dannyvankooten** [comment]
Hello @frietboer,

Can you please elaborate on what exactly you would like to achieve, as I'm not fully understanding it yet. 

Currently, the "Top pages" table in the bottom left of the dashboard shows the top pages across all post types. Do I understand correctly that you want to have a separate table per post type?

**Comment by frietboer** [comment]
We have a site with a lot of content, containing multiple post types. It would be nice to be able to view the list of most viewed posts per post type. The shortcode would be great for this, if only it would be able to display the actual number of views/visitors. If this is not feasable, I will probably create a custom wp_query loop and add the koko_analytics_counter shortcode.

Other option could be adding filters to the main dashboard view, for filtering the list.
